docs/docs/how-to/cloud/set-node-version.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Set a Node.js Version for Your Site"
+description: "How to set a Node version for your Gatsby Cloud site."
+---
+
+## Introduction
+
+You can specify a Node.js version for site in Gatsby Cloud in two different ways, by environment variable or `.nvmrc` file. If you don't set a Node version, Gatsby cloud will use the following defaults:
+
+- The default Node.js version is 14.
+- If you are using npm 7, the default is 16.
+
+### Environment variable
+
+You can set your Node.js version using the `NODE_VERSION` environment variable inside Gatsby Cloud. Read more about setting environment variables [here](../../reference/cloud/managing-environment-variables).
```

Comment:
"Here" is a bad link text: https://www.gatsbyjs.com/contributing/gatsby-style-guide/#use-clear-hyperlinks

```suggestion
You can set your Node.js version using the `NODE_VERSION` environment variable inside Gatsby Cloud. Read more about [setting environment variables](/docs/reference/cloud/managing-environment-variables).
```